parquet-column/src/main/java/org/apache/parquet/column/values/plain/PlainValuesWriter.java

```diff
@@ -127,7 +127,6 @@ public void reset() {
   @Override
   public void close() {
     arrayOut.close();
-    out.close();
```

Comment:
`out` in this case is wraps `arrayOut`, and it has no buffering, so it does not need to be explicitly closed, or better, in the future, `out` should be closed only and `arrayOut` will be closed implicitly.  Anyway, I came to this conclusion because this is how `close()` is implemented in a couple of other Writer classes as well.

Reply:
I would either close only `arrayOut` but with comments about what you have written here or close only `out` and catching the `IOException` that would never occur and comment in the catch block similarly.

Reply:
I am fine fixing this in another ticket. What I think is a key in this change is to put the related comments (`out.close` is not required because...) in the code as well as in this review.

Reply:
@gszadovszky OK, finally got back to this. Ha.  I added a comment.  Please consider for merge.